tools/src/py-mkopensource/testdata/agpl-license/dependency_list.txt

```diff
@@ -5,7 +5,7 @@ Home-page: https://github.com/X-I-A/pyagent
 Author: Soral
 Author-email: soral@x-i-a.com
 License: AGPLv3+
-Location: /home/andres/source/production/emissary/tools/src/py-mkopensource/testdata/agpl-license/venv/lib/python3.9/site-packages
+Location: /home/andres/source/production/emissary/tools/src/py-mkopensource/testdata/agpl-license/venv/lib/python3.10/site-packages
```

Comment:
Since this is test data, I'm assuming the actual path other than the python version...more specifically, the fact that it is tide to `andres`'s folder structure doesn't matter for the test.

Reply:
the actual path doesn't matter too much. I didn't have to change it I just did for completeness sakes